PluginGenerator/Resources/Rules/PluginRulesDefinition.java

```diff
@@ -75,7 +77,7 @@ public void define(Context context) {
 
   public static String getRepositoryKeyForLanguage(String languageKey) {
     //return languageKey.toLowerCase() + "-" + KEY;
-	return KEY.toLowerCase();
+	return "roslyn." + KEY.toLowerCase();
```

Comment:
See the emailed spec for a more complete description of what the values should be.
Here, the repository key is prefixed with ".roslyn". All of the other metadata properties are prefixed by *KEY*.